skrifa/src/outline/glyf/mod.rs

```diff
@@ -62,11 +63,16 @@ impl<'a> Outlines<'a> {
         let loca = font.loca(None).ok()?;
         let glyf = font.glyf().ok()?;
         let glyph_metrics = GlyphHMetrics::new(font)?;
-        let has_var_lsb = glyph_metrics
-            .hvar
-            .as_ref()
-            .map(|hvar| hvar.lsb_mapping().is_some())
-            .unwrap_or_default();
+        let var_metrics = match glyph_metrics.hvar.as_ref() {
+            Some(hvar) => {
+                if hvar.lsb_mapping().is_some() {
+                    AvailableVarMetrics::Present
+                } else {
+                    AvailableVarMetrics::MissingSideBearings
+                }
+            }
+            None => AvailableVarMetrics::Missing,
+        };
```

Comment:
IMHO this might still read better (closer to left-to-right top to bottom) as map? - something like `glyph_metrics.hvar.map(|hvar| hvar.lsb_mapping.map(present).unwrap_or(missing sides)).unwrap_or(missing everything)`